packages/api-client/src/components/CodeInput/CodeInput.vue

```diff
@@ -342,7 +342,7 @@ export default {
   </div>
   <div
     v-if="$slots.icon"
-    class="centered-y group-has-[.cm-focused]:z-1 -right-0.25 absolute flex h-full items-center p-1.5">
+    class="centered-y group-has-[.cm-focused]:z-1 absolute right-0 flex h-full items-center p-1.5">
```

Comment:
this was causing an overflow due to the negative position 
cc: @antlio

Reply:
good catch

Reply:
and not needed as no scroll no more